brave/src/main/java/brave/internal/collect/WeakConcurrentMap.java

```diff
@@ -66,6 +69,12 @@
     return target.remove(key);
   }
 
+  /** Iterates over the entries in this map. */
+  @Override
+  public Iterator<Map.Entry<K, V>> iterator() {
```

Comment:
Added an iterator method to match upstream (instead of my previous approach of the `protected` field which stopped being feasible)